src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -232,27 +232,31 @@ private async Task<TimeSpan> DownloadFullDatabaseAsync()
                     var (succeeded, delay) = await DownloadFullDatabaseWorkerAsync().ConfigureAwait(false);
                     if (succeeded)
                     {
-                        await _service._progressService.OnDownloadFullDatabaseSucceededAsync(_service._updateCancellationToken).ConfigureAwait(false);
+                        // Make sure the completion notification is sent even if cancellation is requested
+                        await _service._progressService.OnDownloadFullDatabaseSucceededAsync(CancellationToken.None).ConfigureAwait(false);
                     }
                     else
                     {
+                        // Make sure the completion notification is sent even if cancellation is requested
                         await _service._progressService.OnDownloadFullDatabaseFailedAsync(
-                            EditorFeaturesWpfResources.Downloading_index_failed, _service._updateCancellationToken).ConfigureAwait(false);
+                            EditorFeaturesWpfResources.Downloading_index_failed, CancellationToken.None).ConfigureAwait(false);
```

Comment:
shoudl we just not make these methods cancellable?

Reply:
In theory there are still cases where you would want to cancel, e.g. if VS is shutting down.